gradle.properties

```diff
@@ -16,4 +16,4 @@ opentelemetry.version=1.20.1
 opentracing.version=0.33.0
 
 ## target JDK release version
-jdk.version=8
+jdk.version=17
```

Comment:
is it required to bump the baseline for the full project or can it be just for the spring-boot-starter (which is already done)?

Reply:
Tried it now by only bumping the spring related modules. 

But looks like we are depending on it as well in the test module, which therefore would require a bump to 17 as well.